src/components/PageSections/Header/index.js

```diff
@@ -144,71 +144,74 @@ class Header extends Component {
         })}
         role="banner"
       >
-        <GovSite
-          toggleHowYouKnowMenu={this.toggleHowYouKnowMenu}
-          menuIsOpen={this.state.howYouKnowmenuIsOpen}
-        />
-        <div className="coa-Header__mobile-languages">
-          <LanguageSelectBar path={path} />
-        </div>
-        <div className="coa-Header__container">
-          <div className="coa-Header__controls">
-            <div className="coa-Header__left-controls">
-              <div className="coa-Header__desktop-languages">
-                <LanguageSelectBar path={path} />
+        <div className="coa-Header--container">
```

Comment:
'container' seems like a confusing modifier name for me. at first glance it looks like the purpose of the container is to set a max width and center the content.